app/src/main/java/io/apicurio/studio/auth/AuthorizedInterceptor.java

```diff
@@ -44,6 +45,7 @@ public class AuthorizedInterceptor {
     SecurityIdentity securityIdentity;
 
     @ConfigProperty(name = "quarkus.oidc.tenant-enabled")
+    @Info(category = "auth", description = "OIDC tenant enabled", availableSince = "1.0.0")
```

Comment:
Should probably read:  `OIDC authentication enabled`

Reply:
Hm - if we inject the same property in multiple places (as we do for this one) do we want to `@Info` describe in every place?

Reply:
Do you know what is the difference between this property and `quarkus.oidc.enabled` ("OIDC enabled")?